### DISCUSSION TITLE
Divide performance calculation?

### DISCUSSION THREAD

**Discussion by n3than** [ppy/osu#16660] [Ideas]
As I understand it, current performance calculation is using the summerized stats of a play. Wouldn't it be far more accurate to submit some sort of list/array? Dividing the beatmap in easier/harder parts should give a more accurate evaluation and additional ways to tune the system.

- The main problem with such a solution are old plays that cannot be re-evaluated because of missing data.
- On the contrary, the pp count of top plays shouldn't change, because top plays consist mostly of fcs. 

Do I speak gibberish or is my point valid? Any opinions?

**Comment by bdach** [comment]
not a novel idea, pp devs have been asking for detailed data such as combo run lengths / individual judgement offsets for a long time. as you mention the big issue there is backwards compatibility with past scores where this data is either not easily available or not available at all (due to no replays).

**Reply by n3than** [reply]
As I mentioned, play data of every top play with close to a full combo can be generated. The pp value of a full combo would stay the same. Plays with less than ~95% percent combo which are relevant for the total pp count of a player are the real problem. Here, we could look at how large the combo is and make generous assumptions about where the combo started and where it ended.
